core/services/runs.go

```diff
@@ -26,11 +26,23 @@ func MeetsMinimumPayment(
 
 func validateMinimumConfirmations(run *models.JobRun, taskRun *models.TaskRun, currentHeight *models.Big, txManager store.TxManager) {
 	updateTaskRunConfirmations(currentHeight, run, taskRun)
+
 	if !meetsMinimumConfirmations(run, taskRun, run.ObservedHeight) {
-		logger.Debugw("Pausing run pending confirmations", []interface{}{"run", run.ID.String(), "required_height", taskRun.MinimumConfirmations}...)
```

Comment:
Why remove the report of the required height? I found that pretty useful, when I was working on the fork test.

Reply:
It's on the next line.